src/scf/self_consistent_field.jl

```diff
@@ -170,10 +177,10 @@ Solve the Kohn-Sham equations with a SCF algorithm, starting at ρ.
         ρnext.real
     end
 
-    fpres = solver(fixpoint_map, ρout.real, maxiter; tol=min(10eps(T), tol / 10))
-    # Tolerance is only dummy here: Convergence is flagged by is_converged
+    # Tolerance and maxiter are only dummy here: Convergence is flagged by is_converged
     # inside the fixpoint_map. Also we do not use the return value of fpres but rather the
     # one that got updated by fixpoint_map
+    fpres = solver(fixpoint_map, ρout.real, typemax(Int); tol=0)
```

Comment:
Sure tol=0 is a good idea. I tried that at some point and this caused numerical problems in *some* fixed point solvers, so I changed it to the value it has now. Have you tested with the crop and the custom anderson and the custom simple we have?

Reply:
Well I'll see with the CI. We should just fix those solvers in that case

Reply:
Seems good. I'll merge it.